gateleen-queue/src/main/java/org/swisspush/gateleen/queue/queuing/QueuingHandler.java

```diff
@@ -33,16 +33,16 @@ public static boolean isQueued(HttpServerRequest request) {
 
     private HttpServerRequest request;
```

Comment:
private HttpServerRequest request;
private Vertx vertx;
private RedisProvider redisProvider;

variables could be marked as final.